tests/jiradeps/test_nxdeps.py

```diff
@@ -1,25 +1,20 @@
 # Copyright 2018 TNG Technology Consulting GmbH, Unterföhring, Germany
 # Licensed under the Apache License, Version 2.0 - see LICENSE.md in project root directory
 
-from unittest.mock import MagicMock
 from pytest import fixture
 
 from jiradeps import create_issue_digraph
 
 
 @fixture
-def test_data():
-    class test_story:
-        id = 1
-        key = 'TEST-123'
-        fields = MagicMock()
+def test_data(epics, stories):
 
     return {
-        'TEST-123': [test_story()]
+        epics[0].key: stories
     }
 
 
 def test_trivial_graph(test_data):
```

Comment:
Would be more clear to remove the `test_data` fixture entirely.

Reply:
True, I'll remove it.